js/src/admin/components/AppearancePage.js

```diff
@@ -21,6 +22,15 @@ export default class AppearancePage extends Page {
   view() {
     return (
       <div className="AppearancePage">
+        <div className="AppearancePage-header">
```

Comment:
These headers could probably be extracted out into their own components, which the extension headers could maybe extend?